plugins/solidigm/solidigm-garbage-collection.c

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
Could also add the SPDX identifier to all files.

Reply:
Done!

Reply:
Thanks! If you are not against it, I would change from old (deprecated since 2.0-rc2 https://spdx.org/licenses/)  `GPL-2.0+` to new  `GPL-2.0-or-later`.

Reply:
Done!